GUI/QtQuick/TxInputsSelectedModel.cpp

```diff
@@ -0,0 +1,24 @@
+#include "TxInputsSelectedModel.h"
+
+TxInputsSelectedModel::TxInputsSelectedModel (QObject *parent)
+    : QSortFilterProxyModel (parent)
+{
+   connect (this, &TxInputsSelectedModel::rowsInserted, this, &TxInputsSelectedModel::rowCountChanged);
+   connect (this, &TxInputsSelectedModel::rowsRemoved, this, &TxInputsSelectedModel::rowCountChanged);
+}
+
+bool TxInputsSelectedModel::filterAcceptsRow (int sourceRow, const QModelIndex &sourceParent) const
+{
+   Q_UNUSED(sourceParent)
```

Comment:
You can just put `sourceParent` into comments:
`bool TxInputsSelectedModel::filterAcceptsRow (int sourceRow, const QModelIndex &/*sourceParent*/) const`
or even omit it